js/background.js

```diff
@@ -28,15 +28,17 @@ function Background() {
 
   localStorage['os'] = os;
 
-  /*
-   * Make sure previous versions of the extensions defaults to showing
-   * Answers on Google/Bing
-   */
   if (localStorage['prev_version'] === undefined) {
-    localStorage['zeroclickinfo'] = 'false';
-  } else {
-    if (localStorage['zeroclickinfo'] === undefined) {
-      localStorage['zeroclickinfo'] = 'true';
+    localStorage['prev_version'] = chrome.runtime.getManifest().version;
```

Comment:
@bsstoner this is the same deal as with Safari -- the first week will most probably be skewed.